go/v1/api/note.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/google/logger"
```

Comment:
Can you move the validation of create note request to a separate PR.

This PR should only have the validation library added. Let's keep each PR as small as possible.

Reply:
done